parkings/importers/permit_areas.py

```diff
@@ -0,0 +1,110 @@
+            'gml': 'http://www.opengis.net/gml/3.2',
+        }
+        self.created = 0
+
+    def import_permit_areas(self):
+        response = self._download()
+        permit_areas_dict = None
+
+        if response is not None:
+            PermitArea.objects.all().delete()
+            permit_areas_dict = self._parse_permit_areas(response)
+        else:
+            logger.error("Download failed.")
+            return False
+
+        if permit_areas_dict is not None:
+            self._save_permit_areas(permit_areas_dict)
+
+    @transaction.atomic
+    def _save_permit_areas(self, permit_areas_dict):
+        logger.info('Saving permit areas.')
+
+        for index, permit_area_dict in enumerate(permit_areas_dict):
+            permit_area = PermitArea(**permit_area_dict)
+            permit_area.save()
+            self.created += 1
+
+    def _parse_permit_areas(self, root):
+        members = self._separate_members(root)
+        logger.info('Parsing permit areas.')
+        parsed_members = []
+
+        for index, member in enumerate(members):
+            parsed_members.append(self._parse_member(member))
+
+        return parsed_members
+
+    def _download(self):
+        logger.info('Getting data from the server.')
+
+        try:
+            wfs = WebFeatureService(
+                url='https://kartta.hel.fi/ws/geoserver/avoindata/wfs',
+                version='2.0.0',
+            )
+            response = wfs.getfeature(
+                typename='Asukas_ja_yrityspysakointivyohykkeet_alue',
+            )
+            return etree.fromstring(bytes(response.getvalue(), 'UTF-8'))
+        except Exception:
+            logger.error('Unable to get data from the server.', exc_info=True)
+
+    def _parse_member(self, member):
+        data = member.find(
+            'avoindata:Asukas_ja_yrityspysakointivyohykkeet_alue',
+            self.ns,
+        )
+        identifier = data.find('avoindata:asukaspysakointitunnus', self.ns).text
+        name = data.find('avoindata:alueen_nimi', self.ns).text
+        geom = self._get_polygons(data.find('avoindata:geom', self.ns))
+
+        return {
+            'name': name,
+            'identifier': identifier,
+            'geom': geom,
+        }
+
+    def _get_polygons(self, geom):
+        """
+        Turns the XML containing coordinates into a multipolygon
+        """
+        polygons = []
+
+        for pos in geom.iter('*'):
+            # get leaf nodes. Treat LinearRing and MultiSurface the same way
+            if len(pos) == 0:
+                positions = list(filter(None, pos.text.split(' ')))
+                points = []
+                points_as_pairs = zip(positions[1::2], positions[::2])
+                for latitude, longitude in points_as_pairs:
+                    points.append(Point(float(latitude), float(longitude)))
+                polygons.append(Polygon(points))
+
+        return MultiPolygon(polygons)
```

Comment:
Is this function copied from another importer? If it is, you should rather move it to a common place so that it can be reused rather than copy-pasted.